Sources/CartingCore/Models/TargetBody.swift

```diff
@@ -53,3 +53,23 @@ final class TargetBody {
         self.productType = productType
     }
 }
+
+struct ProductType {
+    private let value: String
+    init(_ value: String) {
```

Comment:
According to Swift Style Guide:
```suggestion
    init(value: String) {
```